pyte/screens.py

```diff
@@ -1043,6 +1048,26 @@ def debug(self, *args, **kwargs):
         By default is a noop.
         """
 
+    def scroll_up(self, lines):
+        """Scroll up `lines` lines."""
```

Comment:
The documentation states that a scroll with no parameters scrolls 1 line. However, an argument of 0 is used to reset mouse tracking (may be xterm specific). In any case, if we get an argument of 0, we don't want to scroll at all. And if we don't get an argument we do want to scroll exactly 1 line. However, looking at the current implementation of `_parser_fsm`, we may be unable to differentiate between receiving no arguments and receiving a 0 as the first argument.

```suggestion
    def scroll_up(self, lines=None):
        """Scroll up `lines` lines.

           :param lines: number of lines to scroll up. defaults to 1 if unspecified.
        """
        lines = 1 if lines is None else lines
```